crypto/objects/objects.txt

```diff
@@ -371,8 +375,10 @@ rsadsi 2 5		: MD5			: md5
 rsadsi 2 6		:			: hmacWithMD5
 rsadsi 2 7		:			: hmacWithSHA1
 
-member-body 156 10197 1 401 : SM3               : sm3
-member-body 156 10197 1 504 : RSA-SM3		: sm3WithRSAEncryption
+sm-scheme 301           : SM2                   : sm2
```

Comment:
This is the same as `sm2p256v1`, is it not?  I misunderstood it for an algorithm OID earlier, rather than a curve...